modules/websearch/lib/search_engine.py

```diff
@@ -4157,6 +4182,10 @@ def rank_records(req, rank_method_code, rank_limit_relevance, hitset_global, pat
     else:
         related_to = pattern
 
+    if rank_method_code == "recommendations":
```

Comment:
A rank method called "recommendations" is too generic, because it can recommend things based on many criterias.

It would be good to invent a new short code e.g. "rr" or "readrank", and set wanted full translation in rnkMETHODNAME table.  We could also use a rank method name derived from "obelix".   I guess I'd prefer something like "readrank", making parallel with "citerank" that we already have.